app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,75 @@
+class OrdersController < ApplicationController
+  include CurrentCart
```

Comment:
do we have this module?

Reply:
https://github.com/2yar/Podranok/blob/adding_cart/app/controllers/concerns/current_cart.rb